openedx/adg/lms/webinars/models.py

```diff
@@ -319,6 +341,7 @@ class WebinarRegistration(TimeStampedModel):
     week_before_mandrill_reminder_id = models.CharField(
         default='', max_length=255, verbose_name=_('Scheduled Week Before Reminder Id On Mandrill'),
     )
+    user_zoom_id = models.CharField(verbose_name=_('User Zoom Id'), max_length=100, blank=True, null=True, )
```

Comment:
As we are storing ids of all the registrants here maybe we should rename it to something like `registrants_ids` and also reconsider the max_length